### PR TITLE
Fix "custom select example" showing this.hasAttribute not a function

### LINKED ISSUE #22513: Example custom select is not keyboard focusable
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#customizing_select_styles

### What information was incorrect, unhelpful, or incomplete?

The example is incomplete.
While the custom select options accept tab key interaction, the custom select itself doesn't appear in the keychain and cannot be activated by keyboard. 
You have to click with a mouse first, then you can use the tab key.

### What did you expect to see?

I expected the custom select to appear in the keychain.
That is, findable and operable via keyboard only.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

Perhaps adding tabindex=0 to the custom select would suffice, but I've not tested.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/select`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/select/index.md
* Last commit: https://github.com/mdn/content/commit/7d86bc540b9aa3d8d4a8d3109667f73d0b3ff701
* Document last modified: 2022-11-02T05:33:58.000Z

</details>

### PR DESCRIPTION
**Pull request by ya-sh-vardh-an**
### Description

I have replaced the arrow function with regular function at line `495` and `507` in `files/en-us/web/html/element/select/index.md` file.

### Motivation

I am currently learning javascript and how to create registration form so, it was great to start learn more related problem. 
( Let me know if the code follows the guidelines. )

### Additional details

When I was using the custom select example it was throwing an error that `this.hasAttribute is not a function onclick`. So, I compared the file with [custom select example](https://github.com/mdn/html-examples/blob/main/custom-select/index.html) file and found that on some lines there were "arrow functions" ( with 'this' keyword ) is being used as compare to the example file where only "regular functions" being used.

### Related issues and pull requests

Relates #22513 

_PS: I also wanted to included the custom select on keychain but it is not working currently, and I haven't updated the [custom select example](https://github.com/mdn/html-examples/blob/main/custom-select/index.html)_

### CODE REVIEW

File: files/en-us/web/html/element/select/index.md [hunk 1]
```diff
@@ -504,7 +504,7 @@ for (const select of selects) {
     }
   };
 
-  div.onkeyup = (event) => {
+  div.onkeyup = function(event) {
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/select/index.md:507]: This doesn't need changing

File: files/en-us/web/html/element/select/index.md [hunk 2]
```diff
@@ -492,7 +492,7 @@ for (const select of selects) {
   div.appendChild(datalist);
   datalist.style.top = `${header.offsetTop + header.offsetHeight}px`;
 
-  div.onclick = (e) => {
+  div.onclick = function(e) {
     if (!multiple) {
       const open = this.hasAttribute("data-open");
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/select/index.md:497]: Could you replace `this` with `div` instead, and stick to arrow functions?

Comment (ya-sh-vardh-an) [reply files/en-us/web/html/element/select/index.md]: Sure!

File: files/en-us/web/html/element/select/index.md (hunk 1 above)
Comment (ya-sh-vardh-an) [reply files/en-us/web/html/element/select/index.md]: Oh yes, sorry my bad.

**Review by Josh-Cena — APPROVED** [review]
Sure, thanks!

**Pull request merged**